### PR TITLE
Center Associates Directory page intro text

### PR DESCRIPTION
**Pull request by moisesnarvaez**
# READY FOR REVIEW

# Summary
- Follow up on https://github.com/SU-SWS/saa_alumni/pull/709

image

# Review By (Date)
- When possible

# Review Tasks

1. Navigate to https://affiliates--stanford-alumni.netlify.app/stanford-associates/directory/
2. Verify the text of the `Intro` component is centered

# Associated Issues and/or People

- https://fullstacklabs.atlassian.net/browse/STAN-288
- @William-Misiaszek

### CODE REVIEW

**Review by William-Misiaszek — APPROVED** [review]
Small Update for text format to be the same as previous site.

**Comment by rebeccahongsf** [comment]
The Associates site is hosted on Drupal so there are styling restrictions on the edible text field which is why the text was not aligned with the Associates widget record count, search field, and data. 

Our suggestion would be to either:
1) Leave it as-is on dev
2) Center not only the text field but also the text itself as well (this can be done by adding `su-text-center`) to match other Alumni Homesite pages that have center text 😃 
image

image

Thoughts, @William-Misiaszek?

**Comment by William-Misiaszek** [comment]
This sounds like a great alternative @rebeccahongsf! Could you or @moisesnarvaez perform this update?

File: src/components/page-types/associatesDirectoryPage/associatesDirectoryPage.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const AssociatesDirectoryPage = (props) => {
             </Container>
           </header>
           {hasRichText(intro) && (
-            <div className={dcnb('su-big-paragraph su-max-w-prose')}>
+            <div className={dcnb('su-big-paragraph su-max-w-prose su-mx-auto')}>
```
Comment (rebeccahongsf) [inline src/components/page-types/associatesDirectoryPage/associatesDirectoryPage.js:39]:
```suggestion
            <div className={dcnb('su-big-paragraph su-max-w-prose su-mx-auto su-text-center')}>
```
@moisesnarvaez would you be able to make this mini update? 😄 
cc: @William-Misiaszek

**Comment by sherakama** [comment]
Who owns the work in this PR? Who is going to finish it?

**Pull request closed without merging**